tests/playbooks/tests_checkpoint_cleanup.yml

```diff
@@ -67,6 +73,9 @@
                   - name: "{{ interface }}"
                     persistent_state: absent
                     state: down
+            - file:
+                dest: /etc/sysconfig/network-scripts/ifcfg-cptstbr
+                state: absent
```

Comment:
Is this also a workaround for some bug? otherwise lgtm.

Reply:
It is the quick cleanup for the initscripts provider. But now that I look at it again, I noticed it is missing the `{{ interface }}` expansion.